### PR TITLE
Wave 1

### PR DESCRIPTION
**Pull request by noglows**
Created a Sinatra site with yesterday's HTML. Added some additional HTML.

### CODE REVIEW

**Comment by kariabancroft** [comment]
Nice job with your git commit history - looks like good level of detail in your messages

File: my-site.rb [hunk 1]
```diff
@@ -3,6 +3,27 @@
...
+
+  get "/blog" do
+    erb :blog
+  end
 
+  get "/projects" do
+    erb :projects
   end
+
+  get "/index" do
```
Comment (kariabancroft) [inline my-site.rb:21]: It's not exactly necessary, but I would put this route up with the `/` route, just so both "index" routes are in the same area within this site file

File: views/about.erb [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+      <td>Age</td>
+      <td>24</td>
+    </tr>
+    <tr>
+      <td>Birthplace</td>
+      <td>Melbourne, Australia</td>
+    </tr>
+    <tr>
+      <td>Other Cities</td>
+      <td>Champagne, IL <br> St.Charles, IL <br> Marblehead, MA <br> Hinsdale, IL <br> Needham, MA <br> Ft. Lauderdale, FL <br> Seattle, WA</td>
```
Comment (kariabancroft) [inline views/about.erb:38]: I would separate these out onto separate lines for readability

**Comment by kariabancroft** [comment]
OMG KEPLER IS SO CUTE!!! <3

File: my-site.rb (hunk 1 above)
Comment (noglows) [inline my-site.rb:21]: Good call. I will move that for Wave 2.

File: views/about.erb (hunk 2 above)
Comment (noglows) [inline views/about.erb:38]: Will change for Wave 2.

**Pull request merged**